subprojects/docs/src/docs/userguide/migration/upgrading_version_7.adoc

```diff
@@ -75,6 +75,11 @@ The legacy `AntlrSourceVirtualDirectory` API has been removed.
 This change affects the `antlr` plugin.
 In Gradle 8.0 and above, use the `AntlrSourceDirectorySet` source set extension instead.
 
+=== Legacy test sources configuration in `idea` plugin
+
+The deprecated `IdeaModule.getTestSourceDirs()` and `IdeaModule.getTestResourceDirs()` were removed in Gradle 8.0.
```

Comment:
```suggestion
The deprecated methods `IdeaModule.getTestSourceDirs()` and `IdeaModule.getTestResourceDirs()` were removed in Gradle 8.0.
```
Suggestion: add an extra word to make this sentence easier to parse. Jumping right into long qualified method names can be difficult for some readers, so prefacing it with a word like "methods" or "classes" can help set reader expectations of what those monospace terms actually mean.